$OpenBSD: patch-libs_net_grl-net-mock_c,v 1.2 2012/11/08 18:56:48 jasper Exp $

From 87a9dda4c5dfcfcf1c533672e342666faaa0c7d5 Mon Sep 17 00:00:00 2001
From: Jasper Lievisse Adriaanse <jasper@humppa.nl>
Date: Thu, 08 Nov 2012 18:45:44 +0000
Subject: Fix compilation with gcc 4.2

grl-net-mock.c: In function 'init_mock_requester':
grl-net-mock.c:206: error: 'for' loop initial declaration used outside C99 mode

--- libs/net/grl-net-mock.c.orig	Thu Nov  8 20:39:45 2012
+++ libs/net/grl-net-mock.c	Thu Nov  8 20:40:52 2012
@@ -153,6 +153,7 @@ void init_mock_requester (GrlNetWc *self)
 {
   char *config_filename = g_strdup (g_getenv (GRL_NET_MOCKED_VAR));
   enable_mocking = FALSE;
+  int i;
 
   if (config_filename == NULL) {
     return;
@@ -203,7 +204,7 @@ void init_mock_requester (GrlNetWc *self)
     } else {
       g_string_append (pattern, "(?:");
 
-      for (int i = 0; parameter_names[i]; ++i) {
+      for (i = 0; parameter_names[i]; ++i) {
         if (i)
           g_string_append (pattern, "|");
 
